q2_alignment/_sina.py

```diff
@@ -0,0 +1,88 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2018, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+__author__ = "Elmar Pruesse"
```

Comment:
We don't typically contain this information as it will be part of the git log for this. Are you opposed to removing this line?

Reply:
No. I'll remove it.

Reply:
fixed in 4bf5306